test/src/main/java/org/axonframework/test/saga/FixtureExecutionResultImpl.java

```diff
@@ -72,6 +75,12 @@
         eventValidator = new EventValidator(eventBus, fieldFilter);
         eventSchedulerValidator = new EventSchedulerValidator(eventScheduler);
         deadlineManagerValidator = new DeadlineManagerValidator(deadlineManager, fieldFilter);
+        onStartRecordingCallbacks = new CopyOnWriteArrayList<>();
+    }
+
+    public void registerStartRecordingCallback(Runnable onStartRecordingCallback) {
```

Comment:
I am missing javadoc on this function entirely.

Reply:
Added docs here in an additional commit.